refarch-integrations/refarch-dms-integration/refarch-dms-integration-fabasoft/refarch-dms-integration-fabasoft-mock-service/src/test/java/de/muenchen/refarch/integration/dms/fabasoft/mock/service/ExampleTest.java

```diff
@@ -15,20 +14,21 @@
 
 @SpringBootTest(classes = FabasoftMockApplication.class, properties = "mock.port=9070", webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @ActiveProfiles("itest")
-public class ExampleTest {
+class ExampleTest {
 
     @Value("${mock.port:9070}")
     private int port;
 
     private LHMBAI151700GIWSDSoap soapClient;
 
     @BeforeEach
-    public void setUp() {
+    void setUp() {
         this.soapClient = FabasoftClienFactory.dmsWsClient("http://localhost:" + port + "/");
     }
 
     @Test
-    public void execute_createProcedure_request() {
+    @SuppressWarnings({ "PMD.UnitTestShouldIncludeAssert", "PMD.UnusedLocalVariable" })
```

Comment:
Why is `PMD.UnusedLocalVariable` needed here? On top, we should remove the `PMD.UnitTestShouldInlcudeAssert` as the test contains an assertion at the end.

Reply:
Done